forgerock-auth/src/main/java/org/forgerock/android/auth/devicebind/ApplicationPinDeviceAuthenticator.kt

```diff
@@ -98,6 +97,8 @@ open class ApplicationPinDeviceAuthenticator(private val pinCollector: PinCollec
             UnRegister()
         } catch (e: UnrecoverableKeyException) {
             UnAuthorize()
```

Comment:
May be provide code comment here about IOException and UnrecoverableKeyException